File: Niveau 2/4 – Fonctions/2 - Deux codes secrets.py
def pwIsSecret(secret, notFirst):
    print("Entrez le code :")
    password = input()
    if password == secret:
        if not(notFirst):
            print("Premier code bon.")
            return True
        else: 
            print("Bravo.")
            return True
    return False

secret = "4242"
secret2 = "2121"
b = False
c = False
while not(b) or not(c):
    if b:
        c = pwIsSecret(secret2, b)
    else:
        b = pwIsSecret(secret, b)

File: Niveau 2/4 – Fonctions/2 - Deux codes secrets.cpp
#include <iostream>
using namespace std;

void mdp(int m)
{
  int password = 0;
  while (password != m)
  {
    cout << "Entrez le code :" << endl;
    cin >> password;
  }
}

int main()
{
  mdp(4242);
  cout << "Premier code bon." << endl;
  mdp(2121);
  cout << "Bravo.";
}
